app/src/main/res/layout/fragment_account.xml

```diff
@@ -79,26 +79,35 @@
                     style="@style/SettingHeader"
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
-                    android:layout_marginStart="@dimen/marginStandardSmall"
-                    android:layout_marginTop="@dimen/marginStandardMedium"
+                    android:paddingBottom="@dimen/marginStandardMedium"
                     android:text="@string/buttonAccountAssociatedEmailAddresses"
-                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintEnd_toStartOf="@id/changeMailboxButton"
                     app:layout_constraintStart_toStartOf="parent"
                     app:layout_constraintTop_toBottomOf="@id/changeAccountButton" />
 
+                <com.google.android.material.button.MaterialButton
+                    android:id="@+id/changeMailboxButton"
+                    style="@style/IconButtonSmall"
```

Comment:
I'd propose to use the style of `IconButton` and force the icon size to `mediumIconSize` to have a bigger target area for the button

Reply:
Did you verify if the string displayed next to the button is at risk of being wrapped to a second line because of this button?

Reply:
Yes, in spanish the string takes 2 lines either way on a pixel.
On a smaller size, german and english also take 2 lines, but reducing the button is not sufficent to put it on one line
On medium size, small button put it on one line